### DISCUSSION TITLE
Team leads -- community, tech, etc

### DISCUSSION THREAD

**Discussion by taylor** [lfn-cnti/bestpractices#125] [Ideas]
I propose changing the Tech Lead role to be Team Lead.

Goal of leads: 
- To increase contributions and improve on existing contributions

Contributions include, but are not limited to:
- technical - eg. research of a topic, extensive use case and best practice documentation, external testing) as well as
- community - eg. representatives from security and chaos groups providing input to CNF WG. community engagement and marketing. 
- facilitating collaboration
- documentation improvements -- grammar, linting (for markdown), consistency of language, using more inclusive language, structure for accessibility
- tooling and process improvements

**Comment by jeffsaelens** [comment]
@iawells Discuss what we need:
- People who can approve pull requests
- Take more time to discuss additional leads

@taylor Purpose: Promote the adoption of could native best practices for networking applications. What are the lower level purposes that drive the need for these roles?

@jeffsaelens I'm down to just figure out who is allowed to merge, and the rest can be sorted later.

General discussion on if we are over complicating this.

**Comment by Smitholi67** [comment]
@taylor @jeffsaelens 
My general view is we are overcomplicating.   I feel we would benefit from having individuals who can lead us on the work that needs to be done.  Tech Leads or Team Leads (name does not matter to me) could, for example,  help to drive the development of our best practices with some overarching guidance coming from the Co-Chairs.  That might mean helping the tech/team leads to identify and then drive the development of specific use cases/user stories that will be useful in bringing out the best practices.  The Tech/Team leads could work with a subset of interested members in our community and then on our Monday meetings give updates as we progress towards completion at each stage of the way.

**Comment by icarocamelo** [comment]
Any further discussion on this?
